hadoop-hdds/docs/content/feature/Quota.md

```diff
@@ -70,6 +70,12 @@ f. If the cluster is upgraded from old version less than 1.1.0, use of quota on
 ### Storage Space level quota
 Storage space level quotas allow the use of units B, KB, MB, GB and TB. Represents how much storage Spaces will be used.
 
+#### Note:
+
+- You cannot set the quota of volumes and buckets in decimals. For example, 1.5 TB.
+
```

Comment:
As per bug, need update CLI help output also

Reply:
Modified the Exception message accordingly